test/scripts/install-kind.sh

```diff
@@ -10,7 +10,7 @@ sudo ln ./_output/local/go/bin/e2e.test /usr/local/bin/e2e.test
 popd
 
 mkdir -p $GOPATH/bin
-wget -O $GOPATH/bin/kind https://github.com/kubernetes-sigs/kind/releases/download/v0.7.0/kind-linux-amd64
+wget -O $GOPATH/bin/kind https://github.com/kubernetes-sigs/kind/releases/download/v0.8.1/kind-linux-amd64
```

Comment:
When I tried to move to v0.8.1 in my IPv6 CI PR, I found that there were 8 or more tests in IPv4 mode with shared gateway mode that fail consistently with v0.8.1 and pass with v0.7.0. Just a heads up.

Reply:
that may explain the failures, it's failing the DNS tests mainly and I can see weird DNS request out of the cluster.
Main difference with 0.8.1 is that KIND has its own network and docker injects its own DNS.
let me remove this and figure out later
Thanks

Reply:
FWIW I saw the same with KIND 0.8+. The flakiness made little sense. Can help troubleshoot that as needed as well. I added support for the named Docker network into e2e so that piece is at least already supported. Ty!